examples/website1/package.json

```
@@ -18,7 +18,6 @@
     "html-webpack-plugin": "^4.0.0-beta.11",
     "serve": "^11.3.0",
     "webpack": "git://github.com/ScriptedAlchemy/webpack.git#fixing-overrides",
-    "webpack-cli": "^3.3.11",
-    "webpack-external-import": "0.0.0-development"
+    "webpack-cli": "^3.3.11"
```

Should this not be using the beta also?

```suggestion
    "webpack-cli": "^4.0.0-beta.3"
```

---

tobias downgraded it :P so apparently not haha